settings.gradle

```diff
@@ -28,3 +28,5 @@ include ':distributions:worker-default', ':distributions:gateway-default', ':dis
 
 include ':extensions:auth:portability-auth-flickr'
 include ':extensions:data-transfer:portability-data-transfer-flickr'
+
+include ':extensions:data-transfer:portability-data-transfer-smugmug'
```

Comment:
Can we hold off on including this in here until the TransferExtension and AuthServiceExtension are complete?

Reply:
Without that line, IntelliJ doesn't recognize portability-data-transfer-smugmug as a module.  Does that matter?

Reply:
Yeah, looks like I need to include the module for development reasons.

Reply:
Oh youre right, sorry I thought this was the settings.gradle for the distributions.